components/camera/vcamera/vcamera.go

```diff
@@ -0,0 +1,210 @@
+//go:build linux
+
+/*
+Package vcamera creates and streams video to virtual V4L2 capture devices on Linux.
```

Comment:
given vcamera is not a component of any sort right now, could we put this into our testutils?